lib/format.js

```diff
@@ -4,7 +4,7 @@ const ruleURI = require('eslint-rule-documentation');
 
 // (editor: Object, message: Object) => Array<Object>
 function selectSolutions(editor, message) {
-	const fix = {message};
+	const {fix} = message;
```

Comment:
Messages have a `fix` property. This was creating a `fix` object with a `message` property

And https://eslint.org/docs/developer-guide/nodejs-api#-lintmessage-type